testing/py3-pillow-simd/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer: Marcel Esser <marcel.esser@gmail.com>
+pkgname=py3-pillow-simd
+_pkgname=Pillow-SIMD
+pkgver=6.0.0
+pkgrel=0
+_pypi_rel=0 # rare pypi "post-release" version used by this project
+pkgdesc="Python Imaging Library SIMD"
+url="https://github.com/uploadcare/pillow-simd"
+arch="x86_64"
+license="PSF"
+depends="py3-olefile freetype jpeg libwebp tiff libpng lcms2 libjpeg-turbo zlib"
```

Comment:
Everything but py3-olefile is already detected automatically, see the build output starting at like 745: https://cloud.drone.io/alpinelinux/aports/11769/2/1



```